cmd/start.go

```diff
@@ -32,5 +32,6 @@ func init() {
 	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The Openshift release image url.")
 	cmd.PersistentFlags().StringVar(&opts.ServingCertFile, "serving-cert-file", opts.ServingCertFile, "The X.509 certificate file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file, or neither.")
 	cmd.PersistentFlags().StringVar(&opts.ServingKeyFile, "serving-key-file", opts.ServingKeyFile, "The X.509 key file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file, or neither.")
+	cmd.PersistentFlags().StringVar(&opts.ClusterProfile, "cluster-profile", opts.ClusterProfile, "Select a different deployment model")
```

Comment:
`start.NewOptions()` should be defaulting this to `default`.  But I'm not seeing that code anywhere.  Am I just missing it?

Reply:
Never mind, found it with the `defaultClusterProfile` stuff.